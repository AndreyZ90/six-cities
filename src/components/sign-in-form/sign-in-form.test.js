import React from 'react';
import TestRenderer from 'react-test-renderer';

import SignInForm from '@/components/sign-in-form/sign-in-form';

const childrenNode = <div className="some-children-node">Inside children node</div>;
const email = `test@gmail.com`;
const password = `qwerty`;

describe(`SignInForm component snapshot`, () => {
  test(`Should correctly render SignInForm component`, () => {
    const tree = TestRenderer.create(
        <SignInForm email={email} password={password} onButtonSubmit={() => {}}>
          {childrenNode}
        </SignInForm>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
