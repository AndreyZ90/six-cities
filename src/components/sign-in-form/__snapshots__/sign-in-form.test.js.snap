// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInForm component snapshot Should correctly render SignInForm component 1`] = `
<form
  action="#"
  className="login__form form"
  method="post"
  onSubmit={[Function]}
>
  <div
    className="some-children-node"
  >
    Inside children node
  </div>
  <button
    className="login__submit form__submit button"
    type="submit"
  >
    Sign in
  </button>
</form>
`;
